docker-compose.yml

```diff
@@ -26,28 +26,14 @@ services:
         - ./backend/.flake8:/app/.flake8:ro
         - ./backend/.coveragerc:/app/.coveragerc:ro
         - ./backend/lib/:/app/lib:ro
+        # Entrypoint
+        - ./backend/config/entrypoint.sh:/entrypoint.sh:ro
       environment:
```

Comment:
You should be able to remove this environment variable.

Reply:
Done.